### PR TITLE
feat: add percentFormatter prop to PieWidgetUI

### PR DESCRIPTION
**Pull request by jantolg**
# Description
Add to central text in PieChartUI the same text formatter

- [Slack thread](https://cartoteam.slack.com/archives/C01B3D52HJB/p1709021888019739?thread_ts=1709021646.201919&cid=C01B3D52HJB)
- Feature

# Acceptance

1. Go to PieWidgetUI
2. Add percentFormatter prop
3. Check the central text and tootltip have the same format as you set

### CODE REVIEW

File: packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js [hunk 1]
```diff
@@ -281,6 +294,7 @@ PieWidgetUI.propTypes = {
   colors: PropTypes.array,
   formatter: PropTypes.func,
   tooltipFormatter: PropTypes.func,
+  percentFormatter: PropTypes.func,
```
Comment (vmilan) [inline packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js:297]: Export the new prop from types `packages/react-ui/src/types.d.ts`

File: packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ function PieWidgetUI({
   data = [],
   formatter,
   tooltipFormatter,
+  percentFormatter,
```
Comment (vmilan) [inline packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js:52]: Does this work also in `ComparativePieWidget`?

File: packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js (hunk 1 above)
Comment (jantolg) [reply packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js:297]: Done!

File: packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js (hunk 2 above)
Comment (jantolg) [reply packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js:52]: No, I didn't implement it in ComparativePieWidget, I could implement this feature in comparative mode, but maybe could be far away of the scope of this feature

**Review by vmilan — APPROVED** [review]
LGTM, let's wait for @VictorVelarde review

File: packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js (hunk 2 above)
Comment (VictorVelarde) [reply packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js:52]: ComparativePie Widget was indeed coming from PS, if I'm not wrong

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Review by menusal — APPROVED** [review]
LGTM - PS

**Pull request merged**